### ISSUE TITLE
Same issue

### ISSUE THREAD

**Issue by umair-khanzada** [epicfaace/jest-bug-1#1]
I am facing the same issue, have you solved this?

**Comment by epicfaace** [comment]
Yes, by following the suggestion here: https://github.com/kulshekhar/ts-jest/issues/494#issuecomment-383532899
